### ISSUE TITLE
How to resolve “Unable to start status bar server” when launching GHUnit Tests from command line after upgrading to XCode 5

### ISSUE THREAD

**Issue by viteinfinite** [gh-unit/gh-unit#128]
Hi all, if you are encountering the following error

**\* Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Unable to start status bar server. Failed to check into com.apple.UIKit.statusbarserver: unknown error code'

when executing GHUnit from command line (GHUNIT_CLI=1) after migrating to XCode 5, I wrote a [blog post](http://www.viteinfinite.com/2013/09/how-to-resolve-unable-to-start-status-bar-server-when-launching-ghunit-tests-from-command-line-after-upgrading-to-xcode-5/) explaining how to deal with it.

Long story short, it looks like the SpringBoard daemon does not get removed after quitting the iOS Simulator.
`launchctl remove 'com.apple.iPhoneSimulator:com.apple.SpringBoard'`
should fix the problem.
